config.js

```diff
@@ -438,10 +438,28 @@
         },
 
         /**
-         * Enable global filters for ALL telemetry requests that support the filter
+         * Enable global filters for ALL telemetry requests that support the filter. 
+         * Telemetry filters modify the 'filter' field in queries to MCWS. 
          * 
+         * key property is required and other options are optional
+         * globalFilters: array, optional - list of global filters to configure. 
+         * * key: string, required. Filter column, e.g. vcid
+         * * name: string, required. Identifier of the filter in the selection window. 
+         * * icon: 'icon-flag', string, icon. Not implemented - potentially icon for minimized filter list. 
```

Comment:
is icon somewhere? if not let's remove it from the docs. If it's something you're wanting, let's create an issue for it.

Reply:
I'm actually not sure what the intention was - it's possible @davetsay intended to have that icon appear in the status bar if a filter was active, or the icon would be in the UI for each filter line? There are a couple of features I'd like to still add to the plugin that I suspect were out of scope to the original implementation - it's possible that icon was something he was working towards implementing and ran out of time/funding from MGSS, similar to singleSelectionThreshold.

Reply:
Going to leave this one in for now since we haven't heard from Dave and it may be useful in the future.

Reply:
I think originally it was to allow for the dev to customize the indicator icon. If its not being used we can probably remove it.